build.gradle

```diff
@@ -32,12 +33,16 @@ plugins {
 }
 
 repositories {
+    mavenLocal()
```

Comment:
This is actually not really needed for publishing to the local Maven repo, isn't it. This would enable us to read locally published artifacts. Why should we add this behaviour now?

Reply:
Ah, good point!